composer.json

```diff
@@ -28,12 +28,13 @@
     },
     "require": {
         "php": "^7.4 || >=8.1",
+        "composer-runtime-api": "^2.2",
```

Comment:
Why wasn't this necessary in #67?

This might pose a problem with adoption in our repositories as we're using Composer v1 in Jenkins builds, I think. Did you try it out yet?

Reply:
I'm OK with merging it to 9.x as there are only BETA releases at the moment.

Reply:
I am aware of the Composer v1 issue regarding jenkins, and I am communicating about this with devops. For testing purposes, it should be sufficient to publish docker image with Composer v2 with special tag, and use it when manually triggering build in jenkins (there is a parameter for source image). We can then mainline it once the repositories are ready (i expect this to be 1 by 1 process). It was not in #67 because I was not aware of way to enforce it at that time.